fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/AtomicMutationIndexMaintainer.java

```diff
@@ -239,4 +239,8 @@ public boolean skipUpdateForUnchangedKeys() {
     // NOTE: It is possible to convert _LONG entries to _TUPLE using something like
     // maintainer1.scan(...).forEach(maintainer2::saveIndexEntryAsKeyValue).
     // This might require multiple scans in transactions if there are too many groups.
```

Comment:
This comment looks out of place, but looks to always have been at the bottom of this file. Perhaps it can be dropped, or kept below your new method

Reply:
First I moved it below, then dropped it. 
Perfectionism.